### PR TITLE
docs(v2): add Deploy with QuandCDN

### PR DESCRIPTION
**Pull request by steveworley**
## Motivation

Hi there! I've been working on a static site hosting service with a bit of a twist, we provide an interface and version control to static pages with immediate publishing/unpublishing workflow support to blend static sites with traditional dynamic content management systems.

We want to support open-source by providing the service for [free to open-source projects](https://www.quantcdn.io//free-open-source).

We are currently using docusaurus to build our docs pages, you can see a working example:

- https://docs.quantcdn.io/

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### CODE REVIEW

File: website/docs/deployment.mdx [hunk 1]
```diff
@@ -397,3 +397,25 @@ echo subdomain.surge.sh > CNAME
...
 
 You can deploy any other changes in the future with the command `surge`.
+
+## Deploying to QuantCDN
+
+## QuantCDN
+
+1. Install [Quant CLI](https://docs.quantcdn.io/docs/cli/get-started)
+
+2. Create a QuantCDN account by [singing up](https://dashboard.quantcdn.io/register)
```
Comment (HonkingGoose) [inline website/docs/deployment.mdx:407]: I think you have a typo here. 😉 
Should probably be: `signing up`.

Comment (steveworley) [reply website/docs/deployment.mdx]: Singing gives you a secret bonus 😛 , thanks for picking that up!

File: website/docs/deployment.mdx [hunk 2]
```diff
@@ -397,3 +397,25 @@ echo subdomain.surge.sh > CNAME
 ```
 
 You can deploy any other changes in the future with the command `surge`.
+
+## Deploying to QuantCDN
+
+## QuantCDN
```
Comment (slorber) [inline website/docs/deployment.mdx:403]:
```suggestion
```

deleting duplicate heading

Comment (slorber) [reply website/docs/deployment.mdx:403]: @steveworley I can't commit to your PR so will wait for you to apply this suggestion

**Comment by slorber** [comment]
LGTM thanks, just remove the duplicate heading

Also please add your doc site to the showcase ;)

https://v2.docusaurus.io/showcase

**Comment by steveworley** [comment]
Thanks for the feedback @slorber! I've updated the PR to add the QuantCDN docs page to the showcase as well :) Please let me know if I can do anything else for the PR.

**Comment by slorber** [comment]
thanks 👍

**Comment by slorber** [comment]
FYI, we are going to clean up our existing Docusaurus docs deployment page in favor of a GitHub discussions category. 

This means your platform won't appear anymore on this page: https://docusaurus.io/docs/deployment

If you want your hosting / CI / CDN platform to remain documented for Docusaurus users, please submit an entry at: https://github.com/facebook/docusaurus/discussions/categories/deployment-platforms

This will be your responsibility to keep it up to date over time.

**Pull request merged**